app/javascript/components/app.jsx

```diff
@@ -52,11 +52,11 @@ const client = new ApolloClient({
   defaultOptions: {
     query: {
       fetchPolicy: 'network-only',
-      errorPolicy: 'all',
+      errorPolicy: 'none',
     },
     watchQuery: {
       fetchPolicy: 'network-only',
-      errorPolicy: 'all',
+      errorPolicy: 'none',
```

Comment:
NIt: I think its important to distinguish this as a separate commit commenting on the mis-configuration of the apollo client for graphql errors. I link to the docs in the commit message would also be nice.

Reply:
Sure, I'll separate it out and link to the docs in the commit message.